src/modules/consumer.js

```diff
@@ -1,7 +1,7 @@
 const parsers = require('./message-parsers');
 const utils = require('./utils');
 
-const loggerAlias = 'bmq:consumer';
+const loggerAlias = 'amq:consumer';
```

Comment:
`amq` is confusing prefix, since we are using `amqp`, let make it `arnav_mq` as prefix